src/test/regress/sql/multi_mx_modifying_xacts.sql

```diff
@@ -207,6 +207,11 @@ CREATE FUNCTION reject_bad_mx() RETURNS trigger AS $rb$
 $rb$ LANGUAGE plpgsql;
 RESET citus.enable_metadata_sync;
 
+BEGIN;
+  SET local client_min_messages TO WARNING;
+  DROP TRIGGER IF EXISTS reject_bad_mx ON objects_mx_1220103;
```

Comment:
Why are these still left around at this point? Is that from the previous run of this test? If so, can we remove them at the end of the test file instead?

Reply:
yeah we can drop them at the end of the file

Reply:
Looked into that little more and I guess we can't do that because the tables seem to have replication_factor > 1 and that's why the same create trigger command (on shard) complains about that the trigger already exists even if we run this test file once.

Though I don't know why this wasn't the case before ..